BLUESPAWN-client/headers/user/CLI.h

```diff
@@ -15,14 +15,14 @@ class CLI : public IOBase {
 
 	const HANDLE input;
 	const HANDLE output;
+
+public:	
```

Comment:
I remember why the constructor was private. I originally intended IOBase modules to use a singleton structure so we wouldn't have any "multiple instances of what should be a singular entity" issues at play. For now, I think it's good though

Reply:
👍